app/components/Link.js

```diff
@@ -17,10 +17,8 @@ export default class Link extends BaseComponent {
   // We need to a routing function to handle actions
   // that send the user to various url targets on site
   goTo() {
-    chrome.tabs.query({ currentWindow: true, active: true }, () => {
-      chrome.tabs.create({ url: `https://www.chess.com/${this.props.slug}` });
-      window.close();
-    });
+    chrome.tabs.create({ url: `https://www.chess.com/${this.props.slug}` });
+    window.close();
```

Comment:
Looks good now, I was just wondering. @martynchamberlin you know what this `window.close()` call does? Could that also be removed as part of my deletion spree? :P